lib/zone.ts

```diff
@@ -952,7 +952,13 @@ const Zone: ZoneType = (function(global: any) {
   const symbolPromise = __symbol__('Promise');
   const symbolThen = __symbol__('then');
 
-  let _currentZoneFrame = new ZoneFrame(null, new Zone(null, null));
+  const symbolRootZoneSpec = '__rootZoneSpec__'
+  var rootZone: Zone = new Zone(null, null);
+  if (global[symbolRootZoneSpec]) {
+    rootZone = <Zone>rootZone.fork(global[symbolRootZoneSpec]);
```

Comment:
I think we should so that we don't clutter global space.
I can't see any reason not to.